frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -1,25 +1,28 @@
 import * as React from "react";
+import { useNavigate } from "react-router-dom";
+
+import { getUrlForFormSectionID } from "app/component/pollingstation/utils";
 
 import {
-  addValidationResultToFormState,
   ApiError,
   ApiResponseStatus,
   Election,
+  getClientState,
   GetDataEntryResponse,
   getInitialFormState,
   getInitialValues,
-  isGlobalValidationResult,
+  getNextSectionID,
   POLLING_STATION_DATA_ENTRY_SAVE_REQUEST_PATH,
   PollingStationResults,
+  SaveDataEntryRequest,
   SaveDataEntryResponse,
+  updateFormStateAfterSubmit,
   useApi,
   useApiGetRequest,
   ValidationResult,
   VotersRecounts,
 } from "@kiesraad/api";
```

Comment:
just noticed randomly, should libs use the @kiesraad alias to themselves? 
I guess only a bit annoying if you ever plan to make it standalone.

Reply:
This happens in more places. We should decide what to do and set up linting for this if possible, what do you think?